Microsoft.DotNet.Interactive/Events/LanguageServiceHoverResponseProduced.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.LanguageService;
+
+namespace Microsoft.DotNet.Interactive.Events
+{
+    public class LanguageServiceHoverResponseProduced : KernelEventBase
```

Comment:
I think calling this `HoverTextProduced` is simpler and at least as informative.